taichi/codegen/spirv/spirv_codegen.cpp

```diff
@@ -516,13 +516,18 @@ class TaskCodegen : public IRVisitor {
   }
 
   void visit(ReturnStmt *stmt) override {
+    // Now we only support one ret
+    const auto &ret_attribs = ctx_attribs_->rets()[0];
+    auto dt = ret_attribs.dt;
+    if (auto tensor_type = dt->cast<TensorType>()) {
+      dt = tensor_type->get_element_type();
+    }
```

Comment:
Shall we get type directly from the `stmt`?

Reply:
fix